keel-core/src/main/kotlin/com/netflix/spinnaker/keel/api/ConstraintState.kt

```diff
@@ -0,0 +1,35 @@
+package com.netflix.spinnaker.keel.api
+
+import java.time.Duration
+import java.time.Instant
+
+data class ConstraintState(
+  val deliveryConfigName: String,
+  val environmentName: String,
+  val artifactVersion: String,
+  val constraintType: String,
+  val status: ConstraintStatus,
+  val createdAt: Instant = Instant.now(),
+  val judgedBy: String? = null,
+  val judgedAt: Instant? = null,
+  val comment: String? = null,
+  val json: String? = null
```

Comment:
What’s this field for? Would we be better off with a generic type if the state requires additional data?